prow/integ-suite-kind.sh

```diff
@@ -39,8 +39,8 @@ export TEST_ENV=kind
 # See https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster
 export PULL_POLICY=IfNotPresent
 
-export HUB=${HUB:-"gcr.io/istio-testing"}
-export TAG="${TAG:-"1.6-alpha.1d3b136325a5592511f72a9b041ea396bdbd5180"}"
+export HUB=${HUB:-"gcr.io/istio-prerelease-testing"}
```

Comment:
these should be coming from prow.

Reply:
comment above about running locally.

Reply:
Have gone back to the daily build as noted above.